tools/cloud-build/daily-tests/builds/slurm-gcp-v6-reconfig-size.yaml

```diff
@@ -31,4 +31,8 @@ steps:
   - |
     set -x -e
     cd /workspace && make
+    BUILD_ID_FULL=$BUILD_ID
```

Comment:
Let's not duplicate it, shortening can be done by python in one place.

Reply:
Moved shortening to `deployment.py`

Reply:
Can we to not add those lines at all, isn't `BUILD_ID` already available to python executable?

Reply:
I tried after removing all the lines, but BUILD_ID is not one of the environment variables. I checked by printing with `printenv` and didn't see it listed. Is there an alternative way to call the variable other than `os.environ.get()`?